packages/components/translations/hds/components/application-state/header/en-us.yaml

```
@@ -0,0 +1 @@
+ERROR: ERROR
```

```suggestion
error: ERROR
```
[Nit] I think this can be lowercase since you're calling it with `hds.application-state.header.error`